### ISSUE TITLE
notification config doesn't apply to udiskie-mount/-umount

### ISSUE THREAD

**Issue by tbm** [coldfix/udiskie#188]
I'm sorry this is not the best bug report because I'm not entirely sure what's going on but I hope you can help identify it.

I use `udiskie-mount -a` and `udiskie-umount -a`.  I get notifications that the disks were mounted/unmounted.  The notifications do not go away automatically and I have to click on them for them to go away.

So today I finally looked at the documentation and saw that you can configure the timeout and the default is 1.5 seconds.  But this isn't working here.  I also changed `~/.config/udiskie/config.yml` and added:

```
notifications:
     timeout: 5
```

(and `-1`) to play around but it didn't help either.

Interestingly, when I launch `udiskie`, it seems to work.  I added a

```
        self._log.debug("timeout:", timeout)
```

in Notify _show_notification()

When I start `udiskie`, I get output from that debug line.  When I run `udiskie-mount` or `udiskie-umount`, I don't get any output and the notifications have no timeout.

Also, when I start `udiskie`, I actually got two notificatins about the disk being mounted: one with the correct timeout and one that has no timeout.

So it seems there are two mechanisms, one that is working and one that is not.

Does what I see make any sense to you?

This is with udiskie 1.7.7 as packaged by Debian.

**Comment by coldfix** [comment]
Hi,

`udiskie-mount` and `udiskie-umount` shouldn't show any notifications when called on their own (except for one line terminal output). Since you get two sets of notifications when running `udiskie`, I suspect, you have something else running that it responsible for notifications. Or could also be another udiskie process running in the background that didn't get the memo when you changed the settings. You can check this case by `ps aux | grep udiskie`.

**Comment by tbm** [comment]
Sorry for wasting your time.  Everything is working as expected.

As you suspected, udiskie was running already.  I forgot that I changed my window manager to start udiskie when I logged in.  And then for some reason I assumed the 1.5 second timeout was the default and that wasn't working.  I just checked the docs again and I'm not sure where my assumption was from since it's clear that it's just an example config.

So, everything is working as expected, thanks for your reply!
